dlls/ff/ff_lualib_team.cpp

```diff
@@ -32,6 +33,29 @@ std::ostream& operator<<(std::ostream& stream, const CTeam& team)
 	return stream << "team" << ":" << const_cast<CTeam&>(team).GetName() << ":" << const_cast<CTeam&>(team).GetTeamNumber();
 }
 
+namespace FFLib
+{
+	// helper function to turn the bitmask into a lua table of teams
+	luabind::adl::object GetAllies(CFFTeam *pTeam)
+	{
+		luabind::adl::object luatblAllies = luabind::newtable(_scriptman.GetLuaState());
+		if (pTeam)

```

Comment:
Unsue what luatblAllies is going to be at this stage but I think you can reduce nesting here too and just return immediately

    if (!pTeam)
    {
        return luatblAbllies;
    }